src/components/FullTags/FullTagPage.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="canvas-size overflow">
+    <div v-for="(image, index) in imageUrls.length" :key="index">
+      <div class="image-size">
+        <img :src="imageUrls[index].url" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import dataFetch from "../../api/dataFetching";
+
+export default {
+  props: {
+    tags: Array,
```

Comment:
probably only one tag? I don't see when multiple tags would make sense for that page

Reply:
Right and done